pulp_ostree/tests/functional/conftest.py

```diff
@@ -0,0 +1,121 @@
+import pytest
+
+from pulp_smash.pulp3.bindings import delete_orphans, monitor_task
+from pulp_smash.pulp3.utils import gen_repo
```

Comment:
One of the rewrite goals is to stop using pulp-smash...
Both delete_orphans (as delete_orphans_pre) and monitor_task are provided by fixtures.
gen_repo is not doing much more than creating a dict with a "name" key.